archivers/star/Portfile

```diff
@@ -1,51 +1,114 @@
+                    \n \
+                    \nThe Schily star archiver provides many advantages over GNU tar or \
+                    BSD tar including a built in 'find', fast fifo, \
+                    selectable cli, accurate sparse files, pattern matcher, sophiticated \
+                    'diff', no name length limitations, the handling of ctimes, atimes \
+                    and mtimes for files, the prevention of the clobbering of more recent \
+                    files during extraction, automatic byte swap (when needed), automatic \
+                    format detect, automatic compression detect, support for ACLs and \
+                    file flags, support for all inode metadata, sophticated error \
+                    control, 'ed' like pattern subsitutions, fast built in -copy mode \
+                    and true incremental dump and restore features. \
+                    \n \
+                    \nExcept for the Schily cpio (or scpio) archiver, all the Schily \
+                    archivers are fully compliant with the POSIX 1003.1-2001 standard! \
+                    \n \
+                    \nThe Schily cpio (or scpio) archiver is fully compliant with the \
+                    previous POSIX 1003.1-1988 standard.  (The cpio archiver \
+                    was dropped from the POSIX 1003.1-2001 standard). \
+                    \n \
+                    \nPLEASE NOTE:  Unlike the pax archiver provided by macOS, \
+                    Schily's pax (or spax) archiver is fully compliant with the POSIX \
+                    1003.1-2001 standard!  In other words, unlike the pax \
+                    archiver provided by macOS, the Schily pax (or spax) archiver will \
+                    actually create and read archives in the POSIX 1003.1-2001 pax \
+                    archive format! \
+                    \n \
+                    \nAlso included is the Schily tartest utility used to test whether a \
+                    tar archive is POSIX compliant and the star_sym utility which \
+                    reads a inode database file from a previous incremental restore via \
+                    the star archiver and prints the database in a human readable format. \
+                    \n \
+                    \nOmitted from the port are the mt, smt and rmt binaries to control \
+                    magnetic tape drives.
+
+homepage            https://cdrtools.sourceforge.net/private/star.html
+master_sites        sourceforge:s-tar
+checksums           sha1    2ee419a1167a54701183b79e75ce3e11de9872a5 \
+                    rmd160  43ef9e17244cbb63da91a1cb56a57d3d8519da30 \
+                    sha256  1b18662ebae6c3509020953d4464e654c7ded4752774de987d8dc307dc43f776 \
+                    size    1154422
 
-depends_build       port:smake
+patchfiles          patch-RULES__rules.prg.diff \
+                    patch-gnutar-manpage.diff
+
+use_bzip2           yes
 
 worksrcdir          star-${version}
```

Comment:
isn't it default value?

Reply:
According the the MacPorts Guide, the default value for worksrcdir is ${distname}.  worksrcdir can be used if the extracted source directory has a different name than the distfile.  I will see if I can eliminate it.  

Right now I am focused on updating this PR to `star` version 1.6.1.  So I will need to rewrite the port and probably extract the needed source code from schily tools, modify the RULES (so it van build) and host the source code just for `star`, version 1.6.1 on my GitHib website.

P.S.  I can build `star` v1.6.1 on my Mac.  I did it by extracting the source files needed for 'star` v1.6.1 from the latest Schily Tools tarball and by replacing the RULES directory with  the RULES directory found in `star` version 1.5.3.  

```
Grinchs-MBP:i386-darwin-clang grinch$ ./star -version
star: star 1.6.1 (x86_64-apple-macosx17.7.0) 2021/07/03

Options: acl-POSIX.1e-draft find fflags remote Linux-xattr

Copyright (C) 1985, 88-90, 92-96, 98, 99, 2000-2021 J?rg Schilling
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

It is too bad that the developer did not update his website for `star` himself to the latest version.  Building Schily Tools and then deleting all the binaries you do not need is just too much work for most people who just want to use `star`.

Reply:
@catap  You were right.  I did not need the `worksrcdir` statement in the Portfile since the default value of `${worksrcdir}` is `${distname}` which equals `${name}-${version}`.   

The `worksrcdir` statement has been deleted from the Portfile.  

Thanks for pointing that out to me.